halium-overlay/etc/init/aethercast.override

```diff
@@ -0,0 +1,3 @@
+# Disable until we've fixed wireless display; this also kept spawning two new
```

Comment:
It likely will not work unless rootfs has this file already.

Reply:
That was indeed turned out to be the case, let's just overlay the `aethercast.conf` for simplicity.